Defs/Ammo/Advanced/66mmThermalBolt.xml

```diff
@@ -70,7 +70,7 @@
       <soundAmbient>MortarRound_Ambient</soundAmbient>
       <flyOverhead>true</flyOverhead>
       <dropsCasings>false</dropsCasings>
-      <gravityFactor>20</gravityFactor> <!-- value intentionally increased to maek the shells land faster -->
+      <gravityFactor>8</gravityFactor> <!-- value intentionally increased to make the shells land faster -->
```

Comment:
Already handled in another PR.

Reply:
I tested it with 8, 10 and 12 values, 8 seemed to be the most fitting in my tests

Reply:
10 is good enough.